src/components/homepage/homepagetemplate.js

```diff
@@ -1,58 +1,68 @@
 import React from 'react';
-import Helmet from 'react-helmet';
-import banner from './banner.svg';
 import PropTypes from 'prop-types';
 
-import { withStyles } from '@material-ui/core/styles';
+import Wrenches from './wrenches';
 
-const StyleHeader = () =>
-	<Helmet>
-		<link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet"/>
-	</Helmet>;
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
Move this line up. We want to group all the imports from libraries in one group and imports from self-defined files in one group.